src/priceHistory/saga.ts

```diff
@@ -55,7 +57,7 @@ export function* fetchTokenPriceHistorySaga({
   }
 }
 
-/** @beta - Exclude from Knip dep check */
+/** @beta - Exclude from Knip unused exports */
```

Comment:
if you never plan to actually use this outside the file, why not just remove the "export"?

Reply:
Removed in e988643a3f991108c7f0af6dc87381c2840a667a!